collect_app/src/main/java/org/odk/collect/android/tasks/InstanceUploaderTask.java

```diff
@@ -108,7 +109,7 @@ public Outcome doInBackground(Long... instanceIdsToUpload) {
             publishProgress(i + 1, instancesToUpload.size());
 
             if (completeDestinationUrl != null) {
-                Analytics.log(AnalyticsEvents.INSTANCE_UPLOAD_CUSTOM_SERVER);
+                Analytics.log(AnalyticsEvents.INSTANCE_UPLOAD_CUSTOM_SERVER, "referrer", referrer);
```

Comment:
There's a limit on how many custom parameters we have, so I think we should use the existing generic "label" here rather than add "referrer".

Reply:
Ahh, that's a good call.

Reply:
Done.